test/functional/sqlsrv/srv_007_login_timeout.phpt

```diff
@@ -38,7 +41,7 @@ do {
             echo "Connection failed at $elapsed secs. Leeway is $leeway sec but the difference is $diff\n";
         } else {
             // The test will fail but this helps us decide if this test should be redesigned
-            echo "$numAttempts\t";
+            echo "$numAttempts: $diff\n";
```

Comment:
I think we could label the output to make it immediately obvious what happens if the test fails in a full test run, otherwise we just get two numbers. Something like `echo "Attempts: $numAttempts, Time difference: $diff\n"`.